### PR TITLE
Adding telemetry for logrotate

### PR DESCRIPTION
**Pull request by kevinclark19a**
## Description

Adding a line to the osinfo telemetry specifying whether or not the VM has logrotoate.

---

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made and Travis.CI is passing.

### Quality of Code and Contribution Guidelines
- [ ] I have read the [contribution guidelines](https://github.com/Azure/WALinuxAgent/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: azurelinuxagent/common/version.py [hunk 1]
```diff
@@ -176,6 +176,20 @@ def get_lis_version():
         # imported in this module or we'd log this too.
         return "Failed"
 
+def has_logrotate():
+    """
```
Comment (pgombar) [inline azurelinuxagent/common/version.py:180]: Remove empty quotes or add comment?

**Review by pgombar — APPROVED** [review]

File: azurelinuxagent/common/version.py [hunk 2]
```diff
@@ -176,6 +176,20 @@ def get_lis_version():
...
+    """
+
+    try:
+        logrotate_version = shellutil.run_command(["logrotate", "--version"]).split("\n")[0]
+        return logrotate_version
+    except shellutil.CommandError:
+        # A non-zero return code means that logrotate isn't present on 
+        # the system; --version shouldn't fail otherwise.
+        return "Absent"
```
Comment (larohra) [inline azurelinuxagent/common/version.py:189]: Define constants for this and use it here (and in the `get_lis_version` function above)?

File: azurelinuxagent/common/version.py [hunk 3]
```diff
@@ -176,6 +176,20 @@ def get_lis_version():
...
+    except Exception:
+        return "Failed"
```
Comment (larohra) [inline azurelinuxagent/common/version.py:191]: Same

File: azurelinuxagent/common/version.py (hunk 1 above)
Comment (narrieta) [reply azurelinuxagent/common/version.py:180]: +1

**Review by narrieta — APPROVED** [review]

**Review by pgombar — APPROVED** [review]

**Pull request merged**